packages/core/__tests__/rest/__unit__/ZosmfRestClient.unit.test.ts

```diff
@@ -54,6 +62,72 @@ describe("ZosmfRestClient tests", () => {
             expect(processedError.additionalDetails).toContain("Username or password are not valid or expired.");
         });
 
+        it("should handle basic auth error with empty string causeErrors using V3_ERR_FORMAT", () => {
+            jest.spyOn(NextVerFeatures, "useV3ErrFormat").mockReturnValue(true);
+            const zosmfRestClient = new ZosmfRestClient(new Session({
+                hostname: "dummy",
+                type: SessConstants.AUTH_TYPE_BASIC,
+                user: "fakeUser",
+                password: "fakePass"
+            }));
+            (zosmfRestClient as any).mResponse = {
+                statusCode: RestConstants.HTTP_STATUS_401
+            };
+            const error: IImperativeError = {
+                msg: "Rest API failure with HTTP(S) status 401",
+                causeErrors: "",
+                additionalDetails: 'Received HTTP(S) error 401 = Unauthorized.\n\n' +
+                    'Host:              some.company.com\n' +
+                    'Port:              1234\n' +
+                    'Allow Unauth Cert: true' +
```

Comment:
Do we think that users may find "Allow Unauth Cert" confusing, because it is the opposite of the `--reject-unauthorized` option?

Reply:
While the "Allow Unauth Cert" text exists in zowe-cli/.../GetJobs module, the same text is produced by the master version of imperative/.../AbstractRestClient. If we want to to alter this text, we will need to open a Git issue against imperative and modify the imperative repo first.

Reply:
I don't feel strongly enough about this to warrant a separate PR on Imperative for it.